RandLAPACK/comps/rl_syps.hh

```diff
@@ -102,22 +100,19 @@ class SYPS : public SymmetricPowerSketch<T, RNG> {
         int call(
             blas::Uplo uplo,
             int64_t m,
-            const std::vector<T>& A,
+            const T* A,
             int64_t lda,
             int64_t k,
-            RandBLAS::RNGState<RNG>& state,
+            RandBLAS::RNGState<RNG> &state,
```

Comment:
You are inconsnistent about where the & goes... you have `T*& ident` below, but here we have `<RNG> &state`

Reply:
I'll file an issue about this so we can address it in a separate PR.